var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -29,12 +29,12 @@ class VtkM(CMakePackage, CudaPackage, ROCmPackage):
 
     version("master", branch="master")
     version("release", branch="release")
-    version("2.0.0-rc1", sha256="4dc018961eeeb3c6bd4e5443d72e79b9d5fb86901e38fb757f74a7a45dbbc0d8")
     version(
-        "1.9.0",
-        sha256="12355dea1a24ec32767260068037adeb71abb3df2f9f920c92ce483f35ff46e4",
-        preferred=True,
+        "2.0.0",
+        sha256="32643cf3564fa77f8e2a2a5456a574b6b2355bb68918eb62ccde493993ade1a3",
+        preferred=True
```

Comment:
The [code style check](https://github.com/spack/spack/actions/runs/4226235249/jobs/7339454427) wants you to keep the `,` on this line.

```suggestion
        preferred=True,
```

In Python it's generally considered good style to keep a comma after the last item in a comma-separated list to prevent errors when adding things to the list.